identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -12,7 +12,11 @@
                 "email-subscription--subscribed" -> subscription.subscribedTo,
                 "email-subscription--first" -> row.isFirst))">
                 @if(subscription.subscribedTo){<input type="hidden" name="emailSubscription[]" value="@subscription.listId" />}
-                <div class="email-subscription__name">@subscription.name</div>
+                <div class="email-subscription__name">@subscription.name
+                    @if(subscription.subheading.isDefined){
+                        <div class="email-subscription__name">@subscription.subheading</div>
```

Comment:
Hmm, both of these should really be `inline` elements, not `block` level, and should have different class names, maybe 
```
<h2 class="email-subscription__name">@subscription.name</h2>

@if(subscription.subheading.isDefined){
  <h3 class="email-subscription__subheading">@subscription.subheading</h3>
}
```